modules/installation-aws-vpc-security-groups.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-government-region.adoc
+// * installing/installing_aws/installing-aws-secret-region.adoc
+// * installing/installing_aws/installing-aws-private.adoc
+// * installing/installing_aws/installing-aws-vpc.adoc
+
+:_content-type: PROCEDURE
+[id="installation-aws-vpc-security-groups_{context}"]
+= Applying existing security groups to the cluster
+By default, the installation program creates and attaches security groups to control plane and compute machines. The rules associated with the default security groups cannot be modified.
+
+If you are deploying an {product-title} cluster to an existing AWS Virtual Private Cloud (VPC), you can apply additional existing AWS security groups (custom security groups) to your control plane and compute machines. Applying custom security groups can help you meet the security needs of your organization, in such cases where you need to control the incoming and outgoing traffic of your control plane and compute machines.
+
+.Prerequisites
+* You have created the security groups in AWS. For more information, see the AWS documentation about working with link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-security-groups.html[security groups].
+* The security groups must be part of the existing VPC to which you are deploying the cluster. The security groups cannot be associated with another VPC.
+* You have an existing `install-config.yaml` file.
+
+.Procedure
+
+. Edit the `compute.platform.aws.additionalSecurityGroupIDs` parameter to specify one or more custom security groups for your compute machines.
+. Edit `controlPlane.platform.aws.additionalSecurityGroupIDs` parameter to specify one or more custom security groups for your control plane machines.
+. Save the file and reference it when deploying the cluster.
+
+.Sample `install-config.yaml` files that specifies custom security groups
+[source,yaml]
+----
+#...
+compute:
+- hyperthreading: Enabled
+  name: worker
+  platform:
+    aws:
+      additionalSecurityGroupIDs:
+        - sg-1
+        - sg-2
+  replicas: 3
+controlPlane:
+  hyperthreading: Enabled
+  name: master
+  platform:
+    aws:
+      additionalSecurityGroupIDs:
+        - sg-3
+        - sg-4
+  replicas: 3
+platform:
+  aws:
+    region: us-east-1
+    subnets:
```

Comment:
@mjpytlak did we want to note that the subnets are required or just let the user figure that out during install?

Reply:
> did we want to note that the subnets are required or just let the user figure that out during install?

@barbacbd I was originally thinking that given an earlier sample file in the topic states that the subnets are required, I would let it be. However - can't hurt. I updated code block with a few annotations. Outside of that, I made your proposed changes. Thanks for those. Please let me know when you have had a chance to look into the Local Zones case.